bundles/org.openhab.binding.amazonechocontrol/src/main/java/org/openhab/binding/amazonechocontrol/internal/handler/EchoHandler.java

```diff
@@ -1083,6 +1103,10 @@ public void updateState(AccountHandler accountHandler, @Nullable Device device,
                 updateTextToSpeech = false;
                 updateState(CHANNEL_TEXT_TO_SPEECH, new StringType(""));
             }
+            if (updateTextCommand) {
+                updateTextCommand = false;
+                updateState(CHANNEL_TEXT_COMMAND, new StringType(""));
```

Comment:
Probably should make the same changes elsewhere as well.
```suggestion
                updateState(CHANNEL_TEXT_COMMAND, StringType.EMPTY);
```